### PR TITLE
fix #73712 usersdrive.com

### LINKED ISSUE #73712: usersdrive.com
### Issue URL (Ads)
 [https://usersdrive.com/q14guop35u1c.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fusersdrive.com%2Fq14guop35u1c.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 13
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Dutch
Custom filter subscriptions: | https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt,<br>https://adblock-thai.github.io/thai-ads-filter/annoyance.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#73712

Pop-up ads with AdGuard App. This rule can block pop-ups for AdGuard App.

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden Hi. Come to this PR, I want to discuss with you something related.

**Comment by wout-codes** [comment]
@zloyden What do you think of making this rule `shortlinkpay.com$$script[tag-content="zoneSett"]` a general rule?

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+usersdrive.com##a[onclick^="window.open("]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:257]: This `usersdrive.com##a[onclick^="window.open("][rel^="noreferrer"]` is more accurate, please change it.

File: EnglishFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -3972,7 +3972,8 @@ neowin.net#$#div[class^="banner-"] { height: 0!important; min-height: 0!importan
 !-------------------
 !
 ! https://github.com/AdguardTeam/AdguardFilters/pull/73469#issuecomment-769622353
-shortlinkpay.com$$script[tag-content="zoneSett"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/73712
+usersdrive.com,shortlinkpay.com$$script[tag-content="zoneSett"]
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:3976]: Ok, let's try it. Remove domains from the rule.
If something goes wrong, I'll revert it.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:3976]: Yes, Thanks for keeping my suggestion.

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: I am not good in ads so yeah.

File: EnglishFilter/sections/general_extensions.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt]: I will also inform you if anything goes wrong.

**Review by wout-codes — COMMENTED** [review]
I will also inform you if anything goes wrong.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
It's too early to make it a generic, but on the other hand, I've seen similar sites.
I also added the comments.

**Pull request merged**